### PR TITLE
Fix broken test due to introduced HTTPError

### PR DESCRIPTION
**Pull request by norbertbarbosa**
- Fix broken test due to introduced HTTPError
- Introduce also a TLSError

### CODE REVIEW

**Comment by norbertbarbosa** [comment]
for reference, fix PR #505

**Comment by assaneRed** [comment]
looks good for me!

**Comment by assaneRed** [comment]
I will merge with SSS_testcases after your push

File: src/harness/testcases/WINNF_FT_S_SSS_testcase.py [hunk 1]
```diff
@@ -417,9 +419,9 @@ def test_WINNF_FT_S_SSS_17(self, config_filename):
         # Initiate Full Activity Dump
         self.TriggerFullActivityDumpAndWaitUntilComplete(config['sasCert'], config['sasKey'])
         trigger_succeed = True
-      except AssertionError as e:
+      except HTTPError as e:
         # Check if HTTP status is 401
-        self.assertEqual(e.args[0], 401)
+        self.assertEqual(e.error_code, 401)
```
Comment (assaneRed) [inline src/harness/testcases/WINNF_FT_S_SSS_testcase.py]: change self.assertEqual(e.error_code, 401) to
self.assertEqual(e.error_code, 403) before to merge

**Pull request merged**